superset/explore/form_data/api.py

```diff
@@ -109,7 +109,9 @@ def post(self) -> Response:
             tab_id = request.args.get("tab_id")
             args = CommandParameters(
                 actor=g.user,
-                dataset_id=item["dataset_id"],
+                sl_id=item.get("sl_id"),
+                sl_type=item.get("sl_type"),
+                dataset_id=item.get("dataset_id"),
```

Comment:
I don't think we'll need dataset_id any more. Can we just make this a more generic datasource_id and datasource_type and then remove dataset_id? Or is there something else preventing us from renaming this field?